Leave/leave_interface.py

```diff
@@ -37,12 +37,14 @@ def AddLeaveRequestToDB(member, message_id, start_date, end_date, leave_type, le
         remaining_emergency_count = GetRemainingEmergencyLeavesCount(member.id)
         annual_leave_balance = leave_db.GetAnnualLeaveBalance(member.id)
         for day in work_days:
-            is_emergency = utils.IsEmergencyLeave(day, leave_type)
-            is_unpaid = utils.IsUnpaidLeave(annual_leave_balance, is_emergency, remaining_emergency_count)
-            if (not (is_unpaid)):
-                annual_leave_balance -= 1
-
-            leave_db.InsertLeave(member.id, message_id, leave_type, day, reason, "", leave_status, is_emergency, is_unpaid)
+            if (leave_type == "Sick"):
+                leave_db.InsertLeave(member.id, message_id, leave_type, day, reason, "", leave_status, False, False)
+            else:
```

Comment:
do elif instead. lets be really explicit here

Reply:
sure

Reply:
Done